OxfordBodleian/Juel-Jensen/BDLaethg28.xml

```diff
@@ -74,45 +81,48 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                 </support>
                                 <extent>
                                     <measure unit="leaf">16</measure>
-                                    <measure unit="leaf" type="blank"></measure>
-                                    <!--<locus target=""/>-->
                                     <dimensions type="outer" unit="mm">
                                         <height>105</height>
+                                        <!--On the metadata is written: 104?-->
```

Comment:
There were corrections carried out to the metadata (my fault :( )
If you go in the "Metadata" folder in our project folder, you will find scans of the latest metadata versions. Sorry for not mentioning this earlier!!